Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_Generic.yml

```diff
@@ -0,0 +1,1063 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 04488cb1-1067-4a7a-8d91-5013b22d86aa
+    type: playbook
+    task:
+      id: 04488cb1-1067-4a7a-8d91-5013b22d86aa
+      version: -1
+      name: Okta - User Investigation
+      description: This playbook performs an investigation on a specific user, using queries and logs from Okta.
+      playbookName: Okta - User Investigation
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    scriptarguments:
+      LoginCountry:
+        complex:
+          root: inputs.LoginCountry
+      UserEmail:
+        complex:
+          root: inputs.UserEmail
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -160,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "77":
+    id: "77"
+    taskid: 03d8328e-95d5-49b2-83c8-c71eac96f6b0
+    type: title
+    task:
+      id: 03d8328e-95d5-49b2-83c8-c71eac96f6b0
+      version: -1
+      name: Cortex XDR
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "68"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1060,
+          "y": 800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "32_28_#default#": 0.1,
+      "32_43_yes": 0.67,
+      "48_1_yes": 0.54,
+      "48_28_#default#": 0.17,
+      "49_28_#default#": 0.18,
+      "49_31_yes": 0.52,
+      "51_28_#default#": 0.17,
+      "51_45_yes": 0.55,
+      "57_28_#default#": 0.38,
+      "57_34_yes": 0.56,
+      "72_28_#default#": 0.1,
+      "72_77_yes": 0.67,
+      "74_28_#default#": 0.14,
+      "74_75_yes": 0.47
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1885,
+        "width": 2140,
+        "x": -160,
+        "y": -140
+      }
+    }
+  }
+inputs:
+- key: SplunkIndex
+  value:
+    simple: '*'
+  required: false
+  description: Splunk's index name in which to search. Default is "*" - All.
+  playbookInputQuery:
+- key: SplunkEarliestTime
+  value:
+    simple: -1d
+  required: false
+  description: The earliest time for the Splunk search query.
+  playbookInputQuery:
+- key: SplunkLatestTime
+  value:
+    simple: now
+  required: false
+  description: The latest time for the Splunk search query.
+  playbookInputQuery:
+- key: UserEmail
+  value: {}
+  required: false
+  description: User Email.
```

Comment:
If the user email is only for OKTA lets specify it. 

The user email to search OKTA logs (for example)

Reply:
Done.